django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -0,0 +1,984 @@
+/* Tablets */
+
+@media (max-width: 991px) {
+    html, body {
```

Comment:
The indentation signals that you're in a media query when you only look at 30 lines in the middle of the file.